src/service/performance-impl.js

```diff
@@ -221,7 +221,7 @@ export class Performance {
    * @private
    */
   setFlushParams_(params) {
-    this.viewer_.setFlushParams(params);
+    this.viewer_.sendMessageCancelUnsent('setFlushParams', params, false);
```

Comment:
I believe the boolean was introduced to reduce the unncessary response postMessage from search viewer. On the other hand, I'm also not sure if search viewer can handle `true` for every message.